Corona-Warn-App/src/deviceForTesters/res/layout/view_crashreport_list_item.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:tools="http://schemas.android.com/tools"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <data>
+
+        <variable
+            name="crashReport"
+            type="de.rki.coronawarnapp.bugreporting.event.BugEvent" />
+
+        <variable
+            name="pos"
+            type="int" />
+    </data>
+
+    <LinearLayout
```

Comment:
Is the extra  LinearLayout necessary for the card style, it is, isn't it?

Reply:
I first did it without it, but had better visual result with it.